app/models/contact-point.js

```diff
@@ -16,7 +16,7 @@ export const CONTACT_TYPE = {
 
 export default class ContactPointModel extends AbstractValidationModel {
   @attr email;
-  @attr telephone;
+  @attr('phone') telephone;
```

Comment:
I think this also means we can simply remove the [`transformPhoneNumbers` util](https://github.com/lblod/frontend-organization-portal/blob/0fc611807a35ff18e07f4188d383801470cad4a1/app/utils/transform-phone-numbers.js)?

Reply:
Yup, seems like it:

![image](https://github.com/user-attachments/assets/eed82829-a9c5-4e35-9821-0d1124e281dd)

It doesn't cause issues though. The backend properly handles this scenario and only keeps a single prefix.